src/routes/teams-admin.js

```diff
@@ -45,8 +45,58 @@ function register(server, options) {
         handler: getAllTeams
     });
 
+    async function getAllTeamsByUser(request, h) {
+        const { query } = request;
+        const user = await User.findOne({
+            where: {
+                id: query.userId
+            },
+            include: [
+                {
+                    model: Team,
+                    attributes: {
+                        exclude: ['deleted']
+                    },
+                    include: [
+                        {
+                            model: User
+                        }
+                    ]
+                }
+            ]
+        });
+        return {
+            list: user.teams.map(({ dataValues }) => {
+                const { user_team, settings, users, ...data } = dataValues;
+                const owner = users.filter(user => user.user_team.team_role === 'owner').pop();
```

Comment:
Since there should only be 1 owner (?) you can use `Array.prototype.find`
```suggestion
                const owner = users.find(user => user.user_team.team_role === 'owner');
```
If that assumption is incorrect, ignore this comment 😅